basic_text_model_WIP.py

```diff
@@ -0,0 +1,89 @@
+import pandas as pd
+from sklearn.pipeline import Pipeline
+from sklearn.feature_extraction.text import CountVectorizer
+from sklearn.linear_model import LogisticRegression
+from sklearn.ensemble import RandomForestClassifier
+from sklearn.metrics import classification_report, ConfusionMatrixDisplay
+from pathlib import Path
+from preprocess_data import split_train_val_test
+import matplotlib.pyplot as plt
+import numpy as np
+
+czech_stop_list = [
+    "být", "v", "a", "sebe", "na", "ten", "s", "z", "že", "který", "o", "mít", "i", "do", "on", "k",
+    "pro", "tento", "za", "by", "moci", "svůj", "ale", "po", "rok", "jako", "však", "od", "všechen",
+    "dva", "nebo", "tak", "u", "při", "jeden", "podle", "Praha", "jen", "další", "jeho", "aby", "co",
+    "český", "jak", "veliký", "nový", "až", "už", "muset", "než", "nebýt", "člověk", "jenž", "léto",
+    "firma", "první", "náš", "také", "my", "jejich", "když", "před", "doba", "chtít", "jiný", "mezi",
+    "ještě", "já", "ani", "cena", "již", "jít", "strana", "či", "druhý", "pouze"
+]
+
+
+def remove_stop_words(input_string: str, stop_list: list = czech_stop_list) -> str:
+    words = input_string.split()
+    filtered_words = [word for word in words if word not in stop_list]
+    output_string = ' '.join(filtered_words)
+    return output_string
+
+
+def load_and_split_data() -> tuple:
+    lemmatized_df = pd.read_csv(Path(__file__).parent / "data" / "lemmatized_dataset.csv")
+    preprocess_df = pd.read_csv(Path(__file__).parent / "data" / "paro_preprocessed.csv")
+
+    lemmatized_df["project_name"] = lemmatized_df["project_name"].astype("string")
+    lemmatized_df["project_description"] = lemmatized_df["project_description"].astype("string")
+    lemmatized_df["public_interest"] = lemmatized_df["public_interest"].astype("string")
+    lemmatized_df["year"] = preprocess_df["year"]
+
+    df_train, df_val, _ = split_train_val_test(lemmatized_df)
+
+    df_train["status"] = df_train["status"].astype("int")
+    df_val["status"] = df_val["status"].astype("int")
+
+    # text columns are joint together because CountVectorizer accepts only one string column
```

Comment:
good approach, but you can try also models using only one column and compare them